kind/kind.sh

```diff
@@ -48,42 +41,18 @@ function _wait_kind_up {
     kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
 }
 
-function _fetch_kind() {
-    mkdir -p "${KIND_DIR}"
-    KIND="${KIND_DIR}"/.kind
-    if [ -f "$KIND" ]; then
-        current_kind_version=$($KIND --version | awk '{print $3}')
-    fi
-    if [[ $current_kind_version != $KIND_VERSION ]]; then
-        echo "Downloading kind v$KIND_VERSION"
-        if [[ "$OSTYPE" == "darwin"* ]]; then
-            curl -LSs https://github.com/kubernetes-sigs/kind/releases/download/v$KIND_VERSION/kind-darwin-${ARCH} -o "$KIND"
-        else
-            curl -LSs https://github.com/kubernetes-sigs/kind/releases/download/v$KIND_VERSION/kind-linux-${ARCH} -o "$KIND"
-        fi
-        chmod +x "$KIND"
-    fi
-}
-
 function _prepare_config() {
-    echo "Building manifests..."
+     # replace registry name in kind.yml
+     sed -i "s/kind-registry/${KIND_REGISTRY_NAME}/g" ${KIND_DIR}/kind.yml
```

Comment:
@jiere `sed -i` will not work on mac. The script fails with this change on mac

Reply:
Yes, I did see @sthaha 's suggestion above, let me change a new version and force-push diff again.